Makefile

```
@@ -106,3 +106,9 @@ ethtests: ${ETHTEST_TAR}
 	tar -xzmf "$<" --strip-components=1 -C "$@"
 	@cat ${ETHTEST_SHASUM}
 	sha256sum -c ${ETHTEST_SHASUM}
+
+install-solc:
+	brew install solidity
+
+compile-solidity:
```

Maybe call this:

```suggestion
compile-solidity-examples:
```

Just to make it very explicit that it compiles the _examples_.

---

Well, the _tests_ I should say...

---

the best of both worlds, test-examples 😎

---

`compile-solidity-test-examples-that-test-compilation-and-are-actually-example-contracts`

---

🤣🤣🤣